### PR TITLE
Benchs

### PR DESCRIPTION
**Pull request by vphilippon**
@grondinjc @maym2104

### CODE REVIEW

File: benchmarks/temp/code/aa.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import time
+
+for i in range(100):
+  print 'Hello World'
+
+str = raw_input("What is your name?: ")
```
Comment (maym2104) [inline benchmarks/temp/code/aa.py]: As grondinjc pointed me out, str is a class, so maybe we should use another variable name, just to be safe.

File: benchmarks/temp/aa.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import time
 
-for i in range(100)
+for i in range(100):
   print 'Hello World'
 
-str = raw_input("What is your name?: ");
```
Comment (maym2104) [inline benchmarks/temp/aa.py:6]: I remember the " : ", but not the semi colon... :/

Comment (vphilippon) [inline benchmarks/temp/aa.py:6]: Times are rough!

File: benchmarks/temp/code/aa.py (hunk 1 above)
Comment (vphilippon) [inline benchmarks/temp/code/aa.py]: Agreed, will fix

File: benchmarks/benchmarks.py [hunk 3]
```diff
@@ -45,9 +45,7 @@ def __init__(self):
       self.callers.append(chr(ord('a')+i / 26) + chr(ord('a')+i % 26))
 
     for i in range(MAX_FILES):
-      filename = ""
-      for j in range(random.randint(1, 100)):
-        filename += '/' + chr(ord('a')+i / 26) + chr(ord('a')+i % 26)
+      filename = '/' + chr(ord('a')+i / 26) + chr(ord('a')+i % 26)
```
Comment (maym2104) [inline benchmarks/benchmarks.py]: The deleted loop above was to simulate a deepness in the project tree. Is the time of the tasks affected when we remove this?

Comment (vphilippon) [inline benchmarks/benchmarks.py]: Not sure if it affects benchmark time. I'll put it back, but with a reduced random range (less sub dir), what do you think? 
@grondinjc @maym2104

Comment (maym2104) [inline benchmarks/benchmarks.py]: Ok with me.

**Comment by vphilippon** [comment]
Ready for review, again

**Comment by grondinjc** [comment]
Good!

**Pull request merged**